tests/ballerina-unit-test/src/test/resources/test-src/streaming/streamingv2-unique-length-window-test.bal

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/runtime;
+import ballerina/io;
+import ballerina/streams;
+
+type Teacher record {
+    string name;
+    int age;
+    string status;
+    string school;
+};
+
+type TeacherOutput record{
+    string name;
+    int count;
+};
+
+int index = 0;
+stream<Teacher> inputStream;
+stream<TeacherOutput > outputStream;
+TeacherOutput[] globalEmployeeArray = [];
+
+function startUniqueLengthwindowTest1() returns (TeacherOutput[]) {
```

Comment:
We don't need to add enclosing parenthesis for return here since the return is not a tuple.